recipes/capnproto/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.5.3" %}
+
+package:
+  name: capnproto
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/sandstorm-io/capnproto/archive/v{{ version }}.tar.gz
+  md5: 909bd13ad6b8bc840ac78ab8f5bcb0a4
+
+build:
+  number: 0
+  script_env:
+    - MAKEFLAGS
```

Comment:
I would actually love to have `MAKEFLAGS` whitelisted by default in Conda-Build so I can speed-up the `make` execution with `--jobs` parameter on my 64-CPU server without changing a recipe.

I will remove this and file an issue on Conda-Build for that.